### ISSUE TITLE
[question] spark scala or pure python execute optimize and vacuum on Delta Table with no Databricks involvement

### ISSUE THREAD

**Issue by 0xdarkman** [delta-io/delta#1482]
Is it possible to execute optimize and vacuum commands from spark job running in k8s developed in scala or deployment running in k8s developed in pure python?

I don't want to use Databricks or Databricks operators. My end goal is to schedule task in airflow.

So my question is: is it possible?

**Comment by allisonport-db** [comment]
Yes it's possible, please see our documentation for [Optimizations](https://docs.delta.io/latest/optimizations-oss.html) and [Vacuum](https://docs.delta.io/latest/delta-utility.html#remove-files-no-longer-referenced-by-a-delta-table)

**Comment by foss-contributor** [comment]
@0xdarkman, thank you for your contribution! This issue has been inactive for over 18 months and will be closed in one week. To keep it open, please leave a comment on this issue or contact us in the `#contributors` channel on [Delta Lake Slack](https://go.delta.io/slack). We appreciate your involvement in Delta Lake!
